README.md

```diff
@@ -38,7 +38,7 @@ This tools wraps [Factory Configurator Utility (FCU)](https://www.pelion.com/doc
     --- --- ca.crt
     ```
 
-Manually run the validator `./fcu-config-validator.sh` on fcu_config_dir to verify if the respective files are in above structure.
+Manually run the validator `./fcu-config-validator.sh` on fcu_config_dir to verify respective files are in the above structure.
 
 ## Installation
```

Comment:
Is this saying there are two ways to install this tool? One through Docker and one through CLI?

Reply:
We don't list Docker as a prerequisite.